CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Change Log
 
+## [0.23.0](https://github.com/eclipse/lemminx/milestone/36?closed=1) (December 15, 2022)
+
+### Enhancements
+
+ * Codelens, completion, definition, documentLink, highlighting, references, rename support in RNG file. See [#1400](https://github.com/eclipse/lemminx/pull/1400).
+ * Surround selection with XML element. See [#1389](https://github.com/eclipse/lemminx/pull/1389).
+ * Switch experimental formatter to default. See [#1377](https://github.com/eclipse/lemminx/issues/1377).
+ * Full support for `xml.format.maxLineWidth` with experimental formatter. See [#1248](https://github.com/eclipse/lemminx/issues/1248).
+ * Adjust selection range for wrap element when empty selection is in start or end tag. See [#1392](https://github.com/eclipse/lemminx/pull/1392).
```

Comment:
If something is a fix or improvement upon a feature that also made it for the first time into a new release, we usually don't mention the improvement/fix. It's not an improvement from the previous release because it didn't exist, so as far as the user is concerned, it's still part of  the original feature.

For eg. adjusting selection range for wrap element is an improvement on the "Surround selection with XML element", which is new, so no need to mention it again.